### ISSUE TITLE
Feature request: per-file-ignore for specific fixes

### ISSUE THREAD

**Issue by zcutlip** [hhatto/autopep8#572]
---
It would be nice if `autopep8` supported a `per-file-ignores` command line/config option similar to flake8's

There are times when I want most of what autopep8 fixes in my editor, but certain files are problematic (e.g., lots of trailing whitespace). So I want to ignore just those fixes on those specific files.

An example config would like like:

```ini
[autopep8]
per-file-ignores = 
    problem_file_1.py: W291, W391
    problem_file_2.py: E224
```
## Your Environment
* Python version: 3.8.5
* autopep8 version: autopep8 1.5.4 (pycodestyle: 2.6.0)
* Platform: 
ProductName:	Mac OS X
ProductVersion:	10.15.7
BuildVersion:	19H2

**Comment by shaperilio** [comment]
It seems `autopep8` doesn't support in-line comments in the configuration, either. It's confusing that it's set to share the `flake8` section in `setup.cfg` when it can't parse it the way `flake8` does.

**Comment by Avasam** [comment]
Yes please! A lot of PEP8 standard doesn't fit modern type-stubs best practices, I'd like to disable some rules in `*.pyi` only.

**Comment by lucas-labs** [comment]
This would be great.
I usually find that there are certain rules that I just don't want to respect for certain files. Especially, for test files.

When writing tests, we usually do things that we wouldn't do in other scenarios. 
For example, having a long line with a hardocded list of objects to test a function with different inputs. Disabling E501 for `test_*.py` or `conftest.py` files , would be helpful.
